ontology/bmont.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ontology_detail
+layout: ontology_detail
+id: bmont
+title: Biomarker Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: [https://bioportal.bioontology.org/ontologies/BMONT]
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  github: akodamullil
+  label: Alpha Tom Kodamullil
+  orcid: 0000-0001-9896-3531
+description: An application ontology that represents comprehensive knowledge involving a variety of fields of medical and biological aspects.
+domain: health
+homepage: https://github.com/SCAI-BIO/BiomarkerOntology
+in_foundry_order: 
+license:
+  label: CC BY 4.0
+  url: http://creativecommons.org/licenses/by/4.0/
+preferredPrefix: BMONT
+products:
+- id: BMONT.owl
+- id: bmont.obo
+  description: Includes axioms linking to other ontologies, but no imports of those ontologies
+publications:
+- id: 
+  title:
+- id: 
+  title: Using ontologies to describe mouse phenotypes
```

Comment:
Looks like this is a copy from another ontology's metadata record. Remove since there are no associated publications.

```suggestion
```